qiskit/__init__.py

```diff
@@ -38,4 +38,5 @@
 # Import the wrapper, to make it available when doing "import qiskit".
 from . import wrapper
 
-__version__ = '0.6.0'
+with open("VERSION.txt", "r") as version_file:
```

Comment:
There's an issue here: the file VERSION.txt should be placed in the root of the project, as you did!, but this line will try to load the file from the current directory which is `qiskit/`. Let's make this file discovery mechanism a bit more robust and OS independent:
This:
```python
ROOT_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
``` 
will always give you the package rootdirectory, then the only thing you have to do is:
```python
with open(os.path.join(ROOT_DIR, "VERSION.txt"), "r") as version_file:
    __version__ = version_file.readlines()
```
This will address some of the @diego-plan9 comments.